src/themes/ionic.mixins.scss

```diff
@@ -82,6 +82,15 @@
 
 // ---------------------------------------------------------------------------------
 
+// Minimum breakpoint width. Null for the smallest (first) breakpoint.
```

Comment:
Probably incorrect to do this in this PR, but the order of the mixins is giving IDE errors, as we call an undefined method